tests/components/overseerr/test_sensor.py

```diff
@@ -0,0 +1,87 @@
+"""Test the Overseerr sensor platform."""
+from unittest.mock import MagicMock
+
+from homeassistant.components.overseerr.const import DOMAIN
+from homeassistant.core import HomeAssistant, State
+from homeassistant.helpers import entity_registry as er
+
+from tests.common import MockConfigEntry
+
+
+async def test_entity_registry(
```

Comment:
I suggest you use snapshot testing instead and loop over an array with all sensors, would make the test much shorter and readable

Reply:
Snapshot testing has been implemented for entities and states, good suggestion!